<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Save</class>
 <widget class="QWidget" name="Save">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="area">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="import_area">
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MenuButton" name="import_button" native="true">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="preview_area">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="preview_left_area" native="true">
         <property name="minimumSize">
          <size>
           <width>224</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>224</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="MenuButton" name="copy_button" native="true">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MenuButton" name="restore_button" native="true">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MenuButton" name="delete_button" native="true">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="preview_right_area" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="name_label">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="id_label">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="caught_label">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time_label">
            <property name="font">
             <font>
              <family>Liberation Sans</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuButton</class>
   <extends>QWidget</extends>
   <header>src/widgets/menubutton.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
